emulators/dosbox-staging/Portfile

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem            1.0
+PortGroup             github 1.0
+PortGroup             meson 1.0
+
+github.setup          dosbox-staging dosbox-staging 0.78.1 v
+revision              0
+
+categories            emulators
+license               GPL-3+
+maintainers           {@kakuhen} openmaintainer
```

Comment:
please add your (obfuscated) e-mail as well

Reply:
I am in the process of changing e-mails. The one I am currently using for commits will change soon. Once I have finished changing e-mails, I will update all of the ports I maintain to include my e-mail in the obfuscated format.